src/dbcp/extract/acp_projects.py

```diff
@@ -19,7 +50,7 @@ def extract(projects_path: Path) -> dict[str, pd.DataFrame]:
     """
     acp_raw_dfs = {}
     projects_path = dbcp.extract.helpers.cache_gcs_archive_file_locally(projects_path)
-
     acp_raw_dfs["raw_acp_projects"] = pd.read_csv(projects_path)
+    acp_raw_dfs["raw_acp_projects_snapshots"] = _extract_acp_projects_snapshots()
```

Comment:
I think this can replace the "raw_acp_projects" property, where the latest snapshot is the one being used in place of "raw_acp_projects". Do note that the table reading the latest snapshot really should have `_current` in its name according to our naming convention